composer.json

```diff
@@ -18,7 +18,7 @@
   "require": {
     "php": ">=7.3.0",
     "laravel/framework": ">=5.3.0",
-    "phpunit/phpunit": ">=7.0|<=10.0"
+    "phpunit/phpunit": ">=7.0|<10.0"
```

Comment:
```suggestion
    "phpunit/phpunit": ">=7.0"
```

Reply:
@DenTray We cannot remove this restriction, because we will have code that depends on code that does not exist in version 10 of PhpUnit lib.

Reply:
@vitgrams as we discussed, let's add this restriction after the swagger 3 will be released